tests/garage/tf/models/test_new_model.py

```diff
@@ -0,0 +1,77 @@
+import pickle
+
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.models.gaussian_mlp_model import GaussianMLPModel
```

Comment:
you should implement your own simple model in this test, rather than conflating the test with GaussianMLPModel. unit tests should be self-contained.